cli/include/external-cli/ignition/utils/cli/Error.hpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2021 Open Source Robotics Foundation
+ * Copyright (C) 2022 Open Source Robotics Foundation
```

Comment:
We shouldn't change the copyright years, they represent the year the original file was first created

Reply:
Will be hard to automate it, I'll add a user warning to remind us to double check and unstage any unecessary copyright changes if there are any!

Reply:
I went down a deep rabbit hole. But now every copyright year that's changed will be programmatically unpatched!

https://github.com/ignition-tooling/release-tools/pull/712/commits/971f5c6d097ac324815be32401ab7e3e44a34caa

Reply:
Automatically updated file (Error.hpp): https://github.com/ignitionrobotics/ign-utils/blob/545ff6592d60edfdacc57593c96f58f9f8dd186e/cli/include/external-cli/ignition/utils/cli/ConfigFwd.hpp